EIPS/eip-2387.md

```diff
@@ -67,6 +67,10 @@ This fork would give us time to address the community to understand their priori
 
 Muir Glacier never activates on PoA chains – thus will have zero impact on [forkid](https://eips.ethereum.org/EIPS/eip-2124).
 
+### Note on Issuance Reduction
+
+Previous Hardforks to address the Ice Age have also included reductions in the block reward from 5 Eth to 3 Eth to 2 Eth, respectively. In this case, there is no change in issuance, and the block reward remains 2 Eth per block.
```

Comment:
This should be part of EIP-2384, including all the reasoning put into EIP-2387.

EIP-2387 should be a simple meta EIP about the hard fork, similarly to the previous ones.